src/store/config/types.ts

```diff
@@ -54,6 +54,7 @@ export interface GeneralConfig {
   timeformat: string;
   showRateOfChange: boolean;
   flipConsoleLayout: boolean;
+  cameraFullscreenAction: 'embed' | 'rawstream';
```

Comment:
Probably better refactor this to a reusable type (thus avoiding repeating it on CameraItem.vue)